src/Image/index.ts

```diff
@@ -0,0 +1,5 @@
+import Image, { ImageProps, Props } from './Image';
```

Comment:
why are we exporting tcomb `Props` from here? we're not doing in the other components

Reply:
Just addressed the @veej's comment, let me remove it

Reply:
I've amended the last commit and should be fine now!
Anyway, there are other components that export `Props` from `index.ts`, so we need to fix them too

Reply:
whoops... it seems only `BackgroundDimmer` is exporting them.

Reply:
And `ConfirmationInput`